README.md

```diff
@@ -799,16 +530,178 @@ a:hover {
 /* Basic one-line comment */
 ```
 
-<a name="classname-conventions"></a>
-## Classname Conventions
 
-<a name="use-smacss-prefixes-to-distinguish-classes-with-different-roles"></a>
-### Use SMACSS prefixes to distinguish classes with different roles.
 
-**SMACSS** (Scalable and Modular Architecture for CSS) advocates grouping different types
-of classes based on their function. The following is a slightly modified version:
 
-All classnames should have a prefix. Prefixes help insulate new classnames from legacy classnames. The different prefixes are usually a single letter, and are explained in the following section. 
+
+
+
+<a name="object-oriented-css-oocss"></a>
+## Object-Oriented CSS (OOCSS)
+
+OOCSS advocates practices which are beneficial to flexibility, code reuse, maintenance, and performance.
```

Comment:
Is there a good blog post somewhere we can link people to if they want to learn more? I'd rather not have a fuller explanation in line. It's needed, but it could just make this section too long & I'm sure someone else has already written up something super useful and complete that we can reference.